packages/lab/src/date-input/DateInput.css

```diff
@@ -202,7 +202,7 @@
 
 .saltDateInput-startAdornmentContainer > .saltButton,
 .saltDateInput-endAdornmentContainer > .saltButton {
-  --saltButton-padding: calc(var(--salt-spacing-50) - var(--salt-size-border));
+  --saltButton-padding: calc(var(--salt-spacing-50) - var(--salt-size-fixed-100));
```

Comment:
As this relates to padding should it be `salt-spacing-fixed-100`?

Reply:
It's referring to the width of the border (inside border calcuation)